data/planet/cwn/ocaml-weekly-news-02-jul-2024.md

```diff
@@ -0,0 +1,12 @@
+---
+title: OCaml Weekly News, 02 Jul 2024
+description:
+url: https://alan.petitepomme.net/cwn/2024.07.02.html
+date: 2024-07-02T12:00:00-00:00
+preview_image:
+authors:
+- Caml Weekly News
```

Comment:
Interesting these are Caml Weekly News rather than OCaml Weekly News. Is there a reason for that (out of curiosity).